airbyte-webapp/scripts/calculate-source-hash.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/sh
+
+# Script that calculates the hash of all git tracked files as well as the files generated by orval
+
+# Calculate the hash of all files we gitignored in /src/core/request i.e. all the auto generated Orval files.
+GENERATED_CLIENT_HASHES="$(cat .gitignore | grep '/src/core/request' | xargs -I{} openssl sha1 .{})"
```

Comment:
:information_source: Using `openssl sha1` since OSX and Linux have different set of sha tools installed by default, so sha1sum isn't on OSX in the default installation.